indy_node/test/upgrade/test_node_upgrade_in_progress_no_protocol_version.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from indy_common.constants import IN_PROGRESS
+from indy_node.test.helper import TestNode
+from indy_node.test.upgrade.helper import check_node_sent_acknowledges_upgrade
+
+whitelist = ['unable to send message']
+
+
+class TestNodeNoProtocolVersion(TestNode):
```

Comment:
It makes sense to move this class to `indy_node.test.upgrade.helper` module in order not to duplicate it in tests.

Reply:
Agree. Let's do it in a separate PR.